### ISSUE TITLE
Feature request: use environment or global variable to set location

### ISSUE THREAD

**Issue by drmibell** [Steve-Mcl/node-red-contrib-cron-plus#55]
I have home automation flows with solar events that I would like to use at several locations. Editing schedules to change lat/lon can get tedious, so I wonder if a shell environment variable or a global defined in settings.js could be used.

**Comment by Steve-Mcl** [comment]
Have you tried using `${MY_SHELL_ENV_VAR}` ?

**Comment by drmibell** [comment]
Yes, but I got errors because I was using an old version, 1.3.0, I think. Updating fixed it. Only a few small issues. I get a pop-up saying "Error: Cannot get description from node-red" as soon as I start typing in that field, even if everything works after I close the dialog. Also, the node accepts a single string like ${LOCATION}  but not ${LAT} ${LON} or anything else I've tried with two separate variables. (The second is no big deal.)  Finally, because I was running NR on a Pi as a systemd session, I had trouble getting it to pick up my environment variables. I ended up using process.env.LOCATION= "38.897517, -77.036542" in the settings.js file. Is there a better way?
